tools/check_pylint.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+# Copyright 2018-present Samsung Electronics Co., Ltd. and other contributors
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import re
+import subprocess
+
+TOOLS_PATH = os.path.dirname(os.path.abspath(__file__))
+JSREMOTE_ROOT = os.path.join(TOOLS_PATH, '..')
+
+API_PATH = os.path.join(JSREMOTE_ROOT, 'jstest')
+
+PYFILE_REGEX = r'^.*\.py$'
+
+TERMINAL_RED = '\033[1;31m'
+TERMINAL_YELLOW = '\033[1;33m'
+TERMINAL_EMPTY = '\033[0m'
+
+def recursive_check(directory=JSREMOTE_ROOT):
+    '''
+    Check all python files in the given directory recursively.
+    '''
+    exitcode = 0
+    print('%s%s%s' % (TERMINAL_RED, directory, TERMINAL_EMPTY))
+
+    for subdir, _, files in os.walk(directory):
+        for f in files:
+            if re.match(PYFILE_REGEX, f):
+                path_to_check = os.path.join(subdir, f)
+                print('%s%s%s' % (TERMINAL_YELLOW, path_to_check, TERMINAL_EMPTY))
+                exitcode = subprocess.call(['python2', '-m', 'pylint', path_to_check]) or exitcode
```

Comment:
`subprocess.call` should always return with a return code. In this case the `or exitcode` is not necessary.

Reply:
I did it this way so that if one of the pylint checks fails, the rest of the checks will still be run, but the function will not return 0, so Travis will still fail.

Reply:
You are right, I didn't notice the `for` loop.